CHANGELOG.md

```diff
@@ -1,3 +1,267 @@
+  - ACTION REQUIRED: configuration for cert-manager ould now be included under key `cert-manager` in values.yaml file (changed from `certManager`)
+  - ACTION REQUIRED: The chart will no longer configure ClusterIssuers. Refer to the documentation of cert-manager for configuration guide: https://cert-manager.io/docs/configuration/ ([#8392](https://github.com/kubermatic/kubermatic/issues/8392))
+- The old Docker repository at `quay.io/kubermatic/api` will not receive new images anymore, please use `quay.io/kubermatic/kubermatic` (CE) or `quay.io/kubermatic/kubermatic-ee` (EE) instead. This should only affect EE users who were still using the deprecated Helm chart ([#7922](https://github.com/kubermatic/kubermatic/issues/7922))
+- BREAKING: `-dynamic-datacenters` was effectively already always true when using the KKP Operator and has now been removed as a flag. Support for handling a `datacenters.yaml` has been removed ([#7779](https://github.com/kubermatic/kubermatic/issues/7779))
+
+**Dashboard:**
+- Added multiple backup destinations.  
+  ACTION REQUIRED: KKP version 2.19 makes it possible to configure multiple destinations, therefor the current implementation of "backup buckets" will be deprecated soon. Migrate the configuration to a destination to keep using it in the future. It is reachable via the UI Admin Panel -> Backup Destinations ([#3911](https://github.com/kubermatic/dashboard/issues/3911))
+
+### Supported Kubernetes Versions
+- 1.20.13
+- 1.20.14
+- 1.21.8
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster ([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- new endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
+- Manage AKS Clusters natively. Extend external cluster functionality to import existing AKS cluster ([#8244](https://github.com/kubermatic/kubermatic/issues/8244))
+- Create AKS nodePool ([#8399](https://github.com/kubermatic/kubermatic/issues/8399))
+- Upgrade AKS NodePool Kubernetes version  
+  Scale AKS NodePool
+  Get AKS NodePool machineDeployments and nodes
+  Delete AKS NodePool ([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- GET `/providers/eks/clusters` => `/projects/{project_id}/providers/eks/clusters` ([#8423](https://github.com/kubermatic/kubermatic/issues/8423))
+- Adds GET `/api/v2/providers/eks/clusters` ([#8045](https://github.com/kubermatic/kubermatic/issues/8045))
+- GET `/api/v2/providers/aks/clusters` to list Azure AKS clusters ([#8299](https://github.com/kubermatic/kubermatic/issues/8299))
```

Comment:
same comment as above, modified to `/api/v2/projects/{project_id}/providers/aks/clusters`